test/verify/machineslib.py

```diff
@@ -37,6 +37,29 @@ def readFile(name):
     return content
 
 
+# Preparations for physical disk storage pool
+def prepareDiskPool(m):
+    m.add_disk("50M", serial="DISK1")
+    wait(lambda: m.execute("ls -l /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_DISK1"))
+    m.execute("parted /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_DISK1 mklabel gpt")
+
+
+# Preparations for iscsi storage pool
```

Comment:
Same this prepares the iscsi targe and initiator, it does not create the libvirt pool. Either the function name should be changed or the pool create should be moved here.

Reply:
Same as commented above.